visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -378,7 +378,7 @@ describe("codeMapMouseEventService", () => {
 
 				codeMapMouseEventService.onDocumentMouseUp(event)
 
-				expect($rootScope.$broadcast).not.toHaveBeenCalled()
+				expect($rootScope.$broadcast).not.toHaveBeenCalledWith("building-right-clicked")
```

Comment:
You're inside the `onDocumentMouseUP` test. The event is triggered on mosue down.

Reply:
Thanks @NearW. I just fixed failing tests for now. :)